api/src/v1/index.ts

```diff
@@ -0,0 +1,67 @@
+import { Hono } from "hono";
+import { fetchWithZod } from "..";
+import { languagesFilter } from "../utils";
+import { rawFile, versions } from "../versioning";
+
+const app = new Hono();
+
+app.get("/", (c) => c.redirect("/v1/words"));
+
+app.use("/words", languagesFilter(true));
+app.use("/word/:word", languagesFilter(false));
```

Comment:
Inconsistency: 
The lukapona endpoints are `/signs` and `/signs/:sign`, where the word ones are `/words` and `/word/:word` without the plural. Personally, I prefer the former, where we just keep the plural.